apps/condo/domains/news/utils/clientSchema/NewsRenders.tsx

```diff
@@ -34,6 +36,14 @@ type GetTypeRenderType = (intl: IntlShape, search: FilterValue) => GetRenderType
 
 type GetRenderPropertiesType = (intl: IntlShape, search: FilterValue) => GetRenderType
 
+export const ResendButton = styled.div`
```

Comment:
Please try to create resend button as a separate component + CSS module instead of using `styled`.